tests/foreman/cli/test_import.py

```diff
@@ -1338,3 +1404,112 @@ def test_reimport_enable_rh_repos_negative(self, test_data):
             )
         )
+
+    def test_import_content_hosts_default(self):
+        """@test: Import all content hosts from
+        the predefined dataset
+
+        @feature: Import Content-host
+
+        @assert: Profiles for all Content Hosts created
+
+        """
+        for test_data in gen_import_chost_data():
+            with self.subTest(test_data):
+                tmp_dir = self.default_dataset[0]
+                files = dict(self.default_dataset[1])
+                files = update_csv_values(files, test_data, tmp_dir)
+                # import the prerequisities and content hosts
+                imports = import_content_hosts(files, tmp_dir)
+                for entity in imports:
+                    self.assertEqual(imports[entity][0].return_code, 0)
+                # get the sat6 mapping of the imported organizations
+                imp_orgs = get_sat6_id(
+                    Import.csv_to_dataset([files['users']]),
+                    imports['organizations'][1]
+                )
+                # now to check whether all cont. hosts appeared insatellite
+                for imp_org in imp_orgs:
+                    self.assertNotEqual(
+                        ContentHost.list(
+                            {'organization-id': imp_org['sat6']}
+                        ).stdout,
+                        []
+                    )
+
+    def test_reimport_content_hosts_negative(self):
+        """@test: Repetitive Import of all content hosts from
+        the predefined dataset
+
+        @feature: Repetitive Import Content-host
+
+        @assert: Profiles for all Content Hosts created only once
+
+        """
+        for test_data in gen_import_chost_data():
+            with self.subTest(test_data):
+                tmp_dir = self.default_dataset[0]
+                files = dict(self.default_dataset[1])
+                files = update_csv_values(files, test_data, tmp_dir)
+                # import the prerequisities and content hosts
+                imports = import_content_hosts(files, tmp_dir)
+                for entity in imports:
+                    self.assertEqual(imports[entity][0].return_code, 0)
+                # get the sat6 mapping of the imported organizations
+                imp_orgs = get_sat6_id(
+                    Import.csv_to_dataset([files['users']]),
+                    imports['organizations'][1]
+                )
+                chosts_before = [
+                    ContentHost.list(
+                        {'organization-id': imp_org['sat6']}
+                    ).stdout
+                    for imp_org in imp_orgs
+                ]
+                reimport_chosts = Import.content_host_with_tr_data({
+                    u'csv-file': files['system-profiles'],
+                    u'export-directory': tmp_dir,
+                    u'verbose': True
+                })
+                self.assertEqual(reimport_chosts[0].return_code, 0)
```

Comment:
OK, let's go further. So, helper funcs like `content_host_with_tr_data()` return `(result, transition_data)` tuple. As far as i can see `result` was used in tests only to check the `return_code`, right? And, as we agreed, those checks are redundant. So how about i remove returning `result` from helper funcs, and you could you them like 
```python
 for record in ssh_imp_rename:
```
instead of 
```python
 for record in ssh_imp_rename[1]:
```
I can see a lot of such `[1]`'s in code. My only concern is maybe i'm missing something and you used (or planning to use in future tests) that `result` for any purpose, except of checking `return_code`.